src/views/LoginView.vue

```diff
@@ -74,8 +77,54 @@ export default {
     return {
       email: '',
       password: '',
-      errorMessage: '',
-      volodbLogo
+      errorMessageMail: '',
+      errorMessagePassword: '',
+      volodbLogo,
+      errorMail: false,
+      errorPassword: false
+    }
+  },
+  methods: {
+    errorAnimationMail() {
+      this.errorMail = true
+      setTimeout(() => {
+        this.errorMail = false
+      }, 1000)
+    },
+    errorAnimationPassword() {
+      this.errorPassword = true
+      setTimeout(() => {
+        this.errorPassword = false
+      }, 1000)
+    },
+    onSubmit() {
+      this.validate()
+
+      if (this.errorMessagePassword || this.errorMessageMail) {
+        return
+      }
+      try {
+        this.userStore.login(this.email, this.password)
+      } catch (error) {
+        console.log(error)
```

Comment:
if login went wrong: Let the inputs shake again 💃

Reply:
🙏🏼 it works! for now...